### DISCUSSION TITLE
Jetbrains Student pach

### DISCUSSION THREAD

**Discussion by mhmmoshtaghi** [community/community#35529] [GitHub Education]
I created my GitHub Student account and submitted the required documents and my documents were approved. But I can't use Jetbrains Educational Licenses.

**Comment by fadyat** [comment]
Sign in to the JetBrains website using your GitHub account.
It will automatically create a one year license for you.

You can also directly create a license request on the site itself
https://www.jetbrains.com/community/education/#students

**Comment by diva-21** [comment]
Hey @MohammadMoshtagh , as you have applied already, can you try to create using github(student developer pack) option and not with docs submissions.
When u sign with Github , within a min it wil get registered and create a new account in Jetbrains and for student it will provide more explorations

Go to this https://account.jetbrains.com/login

Hope this helps you to get access faster
